### PR TITLE
feat(cli): added additional replica info in replica list

### PR DESCRIPTION
**Pull request by dsavitskiy**

### CODE REVIEW

**Comment by dsavitskiy** [comment]
bors try

**Comment by dsavitskiy** [comment]
bors try

File: io-engine/src/bin/io-engine-client/v1/pool_cli.rs [hunk 1]
```diff
@@ -41,6 +65,16 @@ pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
                 .index(1)
                 .help("Storage pool name"),
         );
+
+    let export = SubCommand::with_name("export")
+        .about("Export storage pool without destroying it")
```
Comment (dsharma-dc) [inline io-engine/src/bin/io-engine-client/v1/pool_cli.rs:70]: Is 'export' some internal operation i.e. developer/test specific? I'm unclear what 'export without destroying' indicates here.

**Review by dsharma-dc — APPROVED** [review]

File: io-engine/src/bin/io-engine-client/v1/pool_cli.rs (hunk 1 above)
Comment (dsavitskiy) [reply io-engine/src/bin/io-engine-client/v1/pool_cli.rs:70]: Export and import are terms from Solaris/ZFS world. In SPDK they call it "unload" and "load".

When a pool is destroyed ("destruct" in SPDK terms), the disk is marked as clean. One is free to create a new pool on it.
When a pool is exported ("unloaded"), the disk is marked as occupied. One cannot create a new pool on it, only "import" ("load") the existing pool.

In v0, we had "create" and "destroy" to manage all the cases, but it proved to be impractical. In v1, we added "import" and "export".

**Review by abhilashshetty04 — APPROVED** [review]

**Review by mtzaurus — APPROVED** [review]

**Comment by dsavitskiy** [comment]
bors merge

**Pull request merged**